python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -112,6 +114,14 @@ class OpRuntimeMetrics:
         default=0, metadata={"map_only": True, "export_metric": True}
     )
 
+    # TODO(mowen): Is this actually different than the wall time already computed
+    # and stored in block exec stats?
+    # Total time spent in data tasks
+    total_data_tasks_time: float = field(default=0, metadata={"export_metric": False})
+
+    # Total time spent in UDFs
+    total_data_udfs_time: float = field(default=0, metadata={"export": False})
```

Comment:
since all the metrics are related to ray data, we can probably omit `data` from the name/output. how about `total_ray_tasks_time` and `total_udfs_time`?